File: projects/organic04/java/RandomNumber.java
public class RandomNumber {
	private int hi, low;
	
	public RandomNumber(int low, int hi) {
		this.low = low;
		this.hi = hi;
	}
	
	public int next() {
		return (int) (Math.random () * (this.hi - this.low) + this.low);
	}
	
	public static int between(int low, int hi) {
		return (int) (Math.random () * (hi - low) + low);
	}
}

File: archives/frontpages/07/php/SCRIPTS/RandomNumber.js
// RandomNumber - a random number generator
function RandomNumber (lo, hi) {
	this.lo = lo;
	this.hi = hi - lo;
	this.init = init;
	this.next = next;
	function next () { return Math.floor (Math.random () * this.hi) + this.lo; }
	function init (lo, hi) { this.lo = lo; this.hi = hi - lo; }
}